src/devices/tuya.ts

```diff
@@ -5451,6 +5451,71 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_k7mfgaen']),
+        model: 'YXZBSL',
+        vendor: 'TuYa',
+        description: 'Smart siren',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [{...tuya.tz.datapoints, key: [...tuya.tz.datapoints.key, 'alarm', 'type', 'volume', 'ringtone', 'duration']}],
```

Comment:
Please add news keys to `tuya.tz.datapoints` directly.

Reply:
done